### PR TITLE
Laboratory work #1, Kostin Egor - 22FPL2

### PR DESCRIPTION
**Pull request by egorkostin24**

### CODE REVIEW

**Comment by artyomtugaryov** [comment]
Nothing to review. any way I hope you will done in time.

File: main.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print("hello world")
```
Comment (artyomtugaryov) [inline main.py:1]: remove the file :(

File: лаба.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+print ("hello world")
```
Comment (artyomtugaryov) [inline лаба.py:1]: remove too

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to remove

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
again - nothing to review. Hope you will be on time

File: лаба.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (artyomtugaryov) [inline лаба.py:1]: remove

File: main.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import string
```
Comment (artyomtugaryov) [inline main.py:1]: remove

File: docs/public/tests.md [hunk 5]
```diff
@@ -8,7 +8,6 @@ To configure tests locally you need to perform several steps:
    `.\venv\Scripts\activate`):
    ```bash
    python -m pip install -r requirements_qa.txt
```
Comment (artyomtugaryov) [inline docs/public/tests.md:10]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: лаба.py (hunk 3 above)
Comment (egorkostin24) [reply лаба.py:1]: done

File: main.py (hunk 4 above)
Comment (egorkostin24) [reply main.py:1]: done

File: docs/public/tests.md (hunk 5 above)
Comment (egorkostin24) [reply docs/public/tests.md:10]: Unfortunately, I haven't understood what exactly you mean by that. I didn't do anything in tests.md file.

Comment (artyomtugaryov) [reply docs/public/tests.md:10]: But I see (and you, I think) that the line has been changed. Please add ``` as it was

File: main.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+def tokenize(text):
```
Comment (artyomtugaryov) [inline main.py:1]: what is it? You must implement functions inside lab_1.../main.py file. Not here

File: лаба.py (hunk 3 above)
Comment (artyomtugaryov) [reply лаба.py:1]: I do not think so. remove the file

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: docs/public/tests.md (hunk 5 above)
Comment (egorkostin24) [reply docs/public/tests.md:10]: done

File: main.py (hunk 6 above)
Comment (egorkostin24) [reply main.py:1]: I think that it's done now :)

File: лаба.py (hunk 3 above)
Comment (egorkostin24) [reply лаба.py]: I am trying to delete it and commit the changes but it's still here..............

Comment (egorkostin24) [reply лаба.py]: fixed!

**Pull request closed without merging**